File: models/Product.js
const mongoose = require('mongoose');

const ProductSchema = new mongoose.Schema({
  productId:String,
  productType: String,
  productName: String,
  productModel: String,
  productBrand: String,
  productPrice: Number,
  additionalDetail: String,
  issuedTo: {
    type: String,
    default: 'NONE',
  },
  status: {
    type: String,
    enum: ['HELD', 'BER', 'ISSUED','UNSERVICEABLE','SERVICEABLE'],
    default: 'HELD',
  },
}, { timestamps: true });

module.exports = mongoose.model('Product', ProductSchema);

File: app/src/main/java/com/example/ncc_inventory/product.kt
package com.example.ncc_inventory

data class Product(
    val productType : String,
    val productId: String,
    val productName: String,
    val productBrand: String,
    val productPrice: String,
    val productModel: String, val additionalDetail: String // Assuming ADT is a string representation of date
)